packages/packages/commonwell-cert-runner/src/document-contribution.ts

```diff
@@ -13,6 +13,8 @@ import {
 import { findOrCreatePatient, findOrCreatePerson } from "./shared-person";
 import { filterTruthy, getEnv, getEnvOrFail } from "./util";
 
+const AXIOS_TIMEOUT = 20_000; // milliseconds
```

Comment:
hypernit: putting the units in vars is preferable, so the devs know at a glance (ie `AXIOS_TIMEOUT_MS` - also the `milliseconds` comment could get lost in future code changes